trunk/source/AbstractInterpretationV2/src/de/uni_freiburg/informatik/ultimate/plugins/analysis/abstractinterpretationv2/domain/relational/octagon/OctInterval.java

```diff
@@ -128,7 +137,11 @@ public IntervalDomainValue toIvlInterval() {
 		if (isBottom()) {
 			return new IntervalDomainValue(true);
 		}
-		return new IntervalDomainValue(mMin.toIvlValue(), mMax.toIvlValue());
+		return new IntervalDomainValue(toIntervalValue(mMin), toIntervalValue(mMax));
+	}
+
+	private static IntervalValue toIntervalValue(final OctValue octValue) {
+		return octValue.isInfinity() ? new IntervalValue() : new IntervalValue(octValue.getValue());
```

Comment:
Sure, I can do that, but this is not really related to SIFA. I just had to replace the constructor `OctValue(IntervalValue)` with this static method, because these classes are now in different plugins.